juniper/src/executor_tests/enums.rs

```diff
@@ -45,28 +44,35 @@ where
 
 fn run_query<F>(query: &str, f: F)
 where
-    F: Fn(&Object) -> (),
+    F: Fn(&Object<DefaultScalarValue>) -> (),
 {
     run_variable_query(query, Variables::new(), f);
 }
 
 #[test]
 fn accepts_enum_literal() {
     run_query("{ toString(color: RED) }", |result| {
-        assert_eq!(result.get_field_value("toString"), Some(&Value::string("Color::Red")));
+        assert_eq!(
+            result.get_field_value("toString"),
+            Some(&Value::string("Color::Red"))
+        );
     });
 }
 
 #[test]
 fn serializes_as_output() {
     run_query("{ aColor }", |result| {
-        assert_eq!(result.get_field_value("aColor"), Some(&Value::string("RED")));
+        assert_eq!(
+            result.get_field_value("aColor"),
+            Some(&Value::string("RED"))
+        );
     });
 }
 
 #[test]
 fn does_not_accept_string_literals() {
-    let schema = RootNode::new(TestType, EmptyMutation::<()>::new());
+    let schema: RootNode<DefaultScalarValue, _, _> =
```

Comment:
These annotations now being required are unfortunate :-/. I guess there is no way to alias the default `DefaultScalarValue` case such that they are not necessary?

Reply:
It is gone. (At least if you use `DefaultScalarValue`)